blog/konferencja-tlumaczy-2024/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Konferencja tłumaczy 2024 - patronujemy!'
+authors: niedzwiedzka
+date: '2024-06-25'
+tags:
+  - 'news'
+  - 'z-kraju'
+  - 'konferencje'
+  - 'tlumaczenia'
+---
+Kolejny rok i kolejna [Konferencja Tłumaczy (KTLC)](https://konferencjatlumaczy.pl/). Już od kilku lat jesteśmy jej dumnym patronem - nie inaczej jest w tym roku. I nie tylko patronem - uczestniczymy w konferencji (czasami nawet jako prelegenci i prelegentki) i możemy ją wam polecić z absolutnym spokojem o wysoką merytoryczną jakość oraz świetną atmosferę. 

```

Comment:
Jesteśmy patronem czy partnerem? Nie wiem czy jest to gdzieś powiedziane oficjalnie i czy robi im to różnicę

Reply:
To jest dobre pytanie. Zazwyczaj jeśli chodzi o konferencje to obejmujemy je patronatem medialnym. Na naszym portalu mamy dwie sekcje - partnerzy i patronat medialny. Partnerzy u nas to raczej organizacje/inicjatywy, z którymi się lubimy i wspieramy. W przypadku tej konferencji, u nich na stronie jesteśmy dodani w sekcji partnerzy, ale oni chyba nie rozróżniają pomiędzy patronatem a partnerstwem. Wg mnie tutaj mówimy o patronacie medialnym, bo to jest event.

Reply:
Poprawić czy nie poprawić, oto jest pytanie.

Reply:
Zostawić jak jest :)